### ISSUE TITLE
rWrapper depends on MASS_7.3-44.tar.gz that cannot be downloaded

### ISSUE THREAD

**Issue by YPares** [NixOS/nixpkgs#11264]
When trying to `nix-env -i nixpkgs.rWrapper` (in order to do [this](https://nixos.org/wiki/R) afterwards), I get a lot of

```
trying http://brieger.esalq.usp.br/CRAN/src/contrib/MASS_7.3-44.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (22) The requested URL returned error: 404 Not Found
```

It tries to find it on various servers but always receives a 404 error.
(I'm using Ubuntu 15.10)

**Comment by FRidh** [comment]
Please see #11230

**Comment by peti** [comment]
This is fixed in `master` and `release-15.09`. The channels will update soon, too.

**Comment by YPares** [comment]
@FRidh @peti I did look for a previous report of this issue... I missed it, sorry for the inconvenience!
